RELEASE_NOTES

```diff
@@ -1,4 +1,35 @@
 
+###############################################################################
+#                                                                             #
+# Trilinos Release 15.0 Release Notes                                         #
+#                                                                             #
+###############################################################################
+
+CMake
+
+  - Change the default for `Trilinos_MUST_FIND_ALL_TPL_LIBS` from `OFF` to
+    `ON`, by setting `Trilinos_MUST_FIND_ALL_TPL_LIBS_DEFAULT` to `ON`. It was
```

Comment:
```suggestion
    `ON`. It was
```

You don't need to tell users how the default was changed.  That is an implementation detail they should not care about.